examples/vue/vue-3/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vue-3",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
+    "preview": "vite preview --port 4173",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
+  },
+  "dependencies": {
+    "@builder.io/sdk-vue": "^0.0.1-55",
+    "vue": "^3.2.37",
```

Comment:
Should possibly be a `peerDependency` (and likely also a `devDependency` for the bundle/build to work). I've not worked on this exact situation for Vue, but in the general case for other similar frameworks the goal is to avoid a library like this bringing in it own copy (or separate version) of the base library, rather to share with the consuming project.

Reply:
Good point! Will do that in a follow up PR